### PR TITLE
article creation

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-00b9b831b8/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-37af10af37/how-to/)

**Comment by stephamon** [comment]
Hey @cyrichardson can you edit this for a full audit?

**Comment by cyrichardson** [comment]
@stephamon Adding this to today's list of todos.

**Comment by cyrichardson** [comment]
Here you go, @stephamon:
[upload-a-file-to-rackspace-email-plus-cloud-drive-cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1674550/upload-a-file-to-rackspace-email-plus-cloud-drive-cyrEdits.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-50dbce4d7d/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-650a9aba19/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ab1ef79cba/how-to/). :bow:

File: content/rackspace-email/upload-a-file-to-rackspace-email-plus-cloud-drive.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+You can confirm that the file has synced to the Cloud Drive by logging in to apps.rackspace.com.
+
+1. Select Cloud Drive from the ribbon at the top of the screen.
+
+2. Select the folder that you saved your document to on your desktop device.
+
+    <img src="{% asset_path rackspace-email/rackspace-email-plus-save-a-file-cloud-drive/webmail_drive.png %}" />
+
+You will see the file that you move to the Cloud Drive. You can now access the file from your other desktop devices that have Cloud Drive installed.
```
Comment (catlook) [inline content/rackspace-email/upload-a-file-to-rackspace-email-plus-cloud-drive.md]: Change to something like: You can see the file that you moved to the Cloud Drive.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d2e1cd7a8d/how-to/). :bow:

**Pull request merged**